test-e2e/fixtures/load-fixture-data.ts

```diff
@@ -0,0 +1,9 @@
+import loadFixture from 'aegir/fixtures'
+import drain from 'it-drain'
+import type { Controller } from 'ipfsd-ctl'
+
+export async function loadFixtureDataCar (controller: Controller, path: string): Promise<void> {
```

Comment:
same as https://github.com/ipfs/helia-verified-fetch/blob/main/packages/interop/src/fixtures/load-fixture-data.ts except for fixture data path